.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "files.exclude": {
+        "**/startupscriptgenerator/bin": true,
+        "**/startupscriptgenerator/pkg": true,
+        "**/startupscriptgenerator/**/vendor": true,
+        "**/startupscriptgenerator/src/github.com": true,
+        "**/startupscriptgenerator/src/gotest.tools": true,
+        "**/startupscriptgenerator/src/golang.org": true
+    },
+    "go.toolsGopath": "${env:HOME}/go"
```

Comment:
When working on Go code in VS Code, it asks to install bunch of Go related tools(linters etc.) which can help us in development process. Since these tools can be use in any Go project (not only Oryx), I made this change so that it installs in a location or workspace other than our Oryx workspace. This way when you want to do `git clean -xdf` you do not unnecessarily clean-up these tools too.